src/mozilla_sec_eia/models/sec_eia_record_linkage/transform_sec_input.py

```diff
@@ -0,0 +1,371 @@
+"""Module for creating an SEC 10K output table with filing companies and subsidiary companies."""
```

Comment:
This creates the SEC output table asset. It does a lot (maybe too much):

* cleaning/preprocessing on the basic 10k and Ex. 21 assets to make them ready to be inputs to the output table
* matches the Ex. 21 subsidiaries to basic 10k filers if that subsidiary also files a 10k
* adds columns like parent_company_cik
* concatenates the 10K filers and Ex. 21 subsidiaries (who don't file a 10k themselves)